CONTRIBUTING.md

```diff
@@ -42,3 +42,16 @@ Any release from master will have a unique version
 ```
 npm run lint
 ```
+
+### Testing
```

Comment:
Not sure if this is the best place to document the testing changes I saw this where the `linting` documented so I thought `testing` would fit in. I can move if needed

Reply:
This is fine for now.